client/session.go

```diff
@@ -50,6 +52,15 @@ func (x ResSessionCreate) PublicKey() []byte {
 	return x.sessionKey
 }
 
+func (x *ResSessionCreate) setExp(exp uint64) {
+	x.exp = exp
+}
+
+// Expiration returns epoch number of the token expiration.
+func (x ResSessionCreate) Expiration() uint64 {
```

Comment:
Why do we need this method? Value doesn't differ with `PrmSessionCreate.SetExp`.